### DISCUSSION TITLE
Lava fog is now visible

### DISCUSSION THREAD

**Discussion by Pow1270** [maruohon/tweakeroo#317] [Q&A]
Now when I got tweakeroo 1.18 I noticed that the fog in lava is back, in the 1.17 version there was no fog in lava (in nether) when "disableNetherFog" was enabled, but in the 1.18 there is. So it doesn't work as it should or lava fog is now a thing. If lava fog is supposed to be visible how do I remove it?

Sorry if my english is hard to understand :/

**Comment by maruohon** [comment]
The `disableNetherFog` feature is not supposed to remove the fog under lava. There is a separate feature for "lava visibility" which tries to slightly balance things by improving the vision based on how good armor enchantments you have and if you have the fire resistance effect (see the tooltip). But I haven't checked under lava in the nether...

At least the `disableRenderDistanceFog` feature used to be broken starting in 1.15 and up to some earlier mod versions in 1.17 such that it would also disable lava fog completely. This is supposed to be fixed in the latest 1.17.1 and 1.18 versions, but I haven't checked it in the nether nor with the `disableNetherFog` feature.

**Reply by Pow1270** [reply]
Thanks for replying so fast, this is very helpful because I saw the "lava visibility" tweak but never understood what it did because I didn`t saw any difference when I enabeld it.

**Reply by maruohon** [reply]
Oh yeah I should mention, these fog options all may be partially or totally broken with Optifine. I think they work with Sodium, but that also isn't guaranteed.

**Reply by Pow1270** [reply]
Ok, I am using sodium so it should be fine
